app-backend/build.sbt

```diff
@@ -11,7 +11,7 @@ lazy val commonSettings = Seq(
   version := "0.0.1-SNAPSHOT",
   scalaVersion := "2.12.8",
   scalafmtOnCompile := true,
-  scapegoatVersion := Versions.ScapegoatVersion,
+  scapegoatVersion in ThisBuild := Versions.ScapegoatVersion,
```

Comment:
Not too satisfied with the conclusion of throwing `in ThisBuild` at it and calling it a day.

We're adding `commonSettings` to each project below, so this shouldn't be needed?

Reply:
I wouldn't sweat this -- I agree that we shouldn't need it, but I don't think there's a good reason to worry about it.